solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -524,12 +584,32 @@ contract RandomBeacon is Ownable {
     function submitRelayEntry(uint256 submitterIndex, bytes calldata entry)
         external
     {
-        relay.submitEntry(
-            submitterIndex,
-            entry,
-            groups.getGroup(relay.currentRequest.groupId)
+        uint256 currentRequestId = relay.currentRequest.id;
+
+        Groups.Group memory group = groups.getGroup(
+            relay.currentRequest.groupId
         );
 
+        (uint32[] memory inactiveMembers, uint256 slashingAmount) = relay
+            .submitEntry(submitterIndex, entry, group);
+
+        punishOperators(inactiveMembers, 2 weeks);
```

Comment:
I think passing it as a parameter is justified as the calling code decides who it punishes and for how long.

Reply:
Makes sense, but it's a bit itchy because a) as for now `punishOperators(inactiveMembers, 2 weeks);` is being triggered only from 1 place, b) isn't `2 weeks` a const and even if we call `punishOperators(inactiveMembers, 2 weeks);` from a different place in our code it would be the same value?

Reply:
This function will be called from DKG as well. Not sure about the value though. If it turns out to be the same we can always extract to const.